extensions/typescript-language-features/package.json

```diff
@@ -202,6 +202,12 @@
             "description": "%typescript.implementationsCodeLens.enabled%",
             "scope": "window"
           },
+          "typescript.useTsgo": {
```

Comment:
Let's mark this as `experimental` for now: `typescript.experimental.useTsgo` and also add `"tags": ["experimental"]`